src/sssom/writers.py

```diff
@@ -317,23 +320,87 @@ def to_fhir_json(msdf: MappingSetDataFrame) -> Dict:
       - ConcpetMap::SSSOM mapping spreadsheet: https://docs.google.com/spreadsheets/d/1J19foBAYO8PCHwOfksaIGjNu-q5ILUKFh2HpOCgYle0/edit#gid=1389897118
 
     TODOs
-    todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
-    TODO: Add additional fields from both specs
+     todo: when/how to conform to R5 instead of R4?: https://build.fhir.org/conceptmap.html
+     TODO: Add additional fields from both specs
      - ConceptMap spec fields: https://www.hl7.org/fhir/r4/conceptmap.html
       - Joe: Can also utilize: /Users/joeflack4/projects/hapi-fhir-jpaserver-starter/_archive/issues/sssom/example_json/minimal.json
-     - SSSOM more fields:
-     - prefix_map
-     - SSSOM spec fields: https://mapping-commons.github.io/sssom/Mapping/
+
```

Comment:
## PyPi updates
@matentzn This is not a code review topic. Just something I need to communicate. I need to have things done by October 20th. This may take iterating a few times, which means I will likely need `sssom-py` released on PyPi multiple times.

I will try to make any PRs short and easy to review. I'm hoping this is something that can be done. If necessary I can branch off of the last release commit/branch and we/I can release patch versions, rather than using the main develop branch.